R/register_extendr.R

```diff
@@ -72,10 +87,16 @@ register_extendr <- function(path = ".", quiet = FALSE, force_wrappers = FALSE,
       outfile = outfile,
       path = path,
       use_symbols = TRUE,
-      quiet = quiet,
-      compile = compile
+      quiet = quiet
     ),
-    error = error_handle
+    error = function(e) {
+      ui_throw(
+        "Failed to generate wrapper functions.",
+        c(
```

Comment:
What is the purpose of the `c()` here? Is it needed? If yes, maybe add a comment explaining what it does?

Reply:
This (likely) was done by me after introducing `ui_*` functions, and I forgot to remove it when the number of additional displayed messages changed.  `c()` can be safely removed in this case.